Kommunicate/Classes/ConversationVCNavBar.swift

```diff
@@ -0,0 +1,172 @@
+        profileImage.widthAnchor.constraint(equalToConstant: 35).isActive = true
+        profileImage.layer.cornerRadius = 18
+        profileImage.clipsToBounds = true
+        
+        statusIconBackgroundColor.bottomAnchor.constraint(equalTo: profileImage.bottomAnchor, constant: 0).isActive = true
+        statusIconBackgroundColor.leadingAnchor.constraint(equalTo: profileImage.trailingAnchor, constant: -10).isActive = true
+        statusIconBackgroundColor.widthAnchor.constraint(equalToConstant: 12).isActive = true
+        statusIconBackgroundColor.heightAnchor.constraint(equalToConstant: 12).isActive = true
+        statusIconBackgroundColor.layer.cornerRadius = 6
+        statusIconBackgroundColor.clipsToBounds = true
+        
+        onlineStatusIcon.centerXAnchor.constraint(equalTo: statusIconBackgroundColor.centerXAnchor).isActive = true
+        onlineStatusIcon.centerYAnchor.constraint(equalTo: statusIconBackgroundColor.centerYAnchor).isActive = true
+        onlineStatusIcon.widthAnchor.constraint(equalToConstant: 10).isActive = true
+        onlineStatusIcon.heightAnchor.constraint(equalToConstant: 10).isActive = true
+        onlineStatusIcon.layer.cornerRadius = 5
+        onlineStatusIcon.clipsToBounds = true
+        
+        profileView.leadingAnchor.constraint(equalTo: profileImage.trailingAnchor, constant: 5).isActive = true
+        profileView.topAnchor.constraint(equalTo: profileImage.topAnchor).isActive = true
+        profileView.bottomAnchor.constraint(equalTo: profileImage.bottomAnchor).isActive = true
+        profileView.trailingAnchor.constraint(equalTo: self.trailingAnchor).isActive = true
+        
+        // Add target to back button.
+        backButton.addTarget(self, action: #selector(backButtonClicked(_:)), for: .touchUpInside)
+    }
+    
+    private func configureBackButton() {
+        if configuration.hideBackButton {
+            backButton.isHidden = true
+        }
+        guard let image = configuration.imageForBackButton else {
+            return
+        }
+        backButton.setImage(image, for: .normal)
+    }
+    
+    private func setupProfileImage(_ contact: ALContact) {
+        let placeHolder = UIImage(named: "placeholder", in: Bundle.kommunicate, compatibleWith: nil)
+        var url: URL?
+        if let imageUrl = contact.contactImageUrl {
+            url = URL(string: imageUrl)
+        }
+        if let downloadURL = url {
+            let resource = ImageResource(downloadURL: downloadURL, cacheKey: downloadURL.absoluteString)
+            self.profileImage.kf.setImage(with: resource, placeholder: placeHolder, options: nil, progressBlock: nil, completionHandler: nil)
+        } else {
+            self.profileImage.image = placeHolder
+        }
+        profileName.text = contact.getDisplayName()
+    }
+    
+    private func setupOnlineStatus(_ contact: ALContact) {
+        if contact.connected {
+            onlineStatusText.text = "Online"
```

Comment:
Can we store `online` and `offline` string in a static struct?
Later it would be easy to modify.